oap-server/server-core/src/main/java/org/apache/skywalking/oap/server/core/storage/model/Model.java

```diff
@@ -37,14 +39,16 @@
     private final boolean record;
     private final boolean superDataset;
     private final boolean isTimeSeries;
+    private final Set<Column.AnalyzerType> analyzer;
 
     public Model(final String name,
                  final List<ModelColumn> columns,
                  final List<ExtraQueryIndex> extraQueryIndices,
                  final int scopeId,
                  final DownSampling downsampling,
                  final boolean record,
-                 final boolean superDataset) {
+                 final boolean superDataset,
+                 final Set<Column.AnalyzerType> analyzer) {
```

Comment:
`List<ModelColumn> columns` should include the analyzer(s) already, we don't need to add this new parameter.